health/micro-ui-react19/web/packages/css/src/pages/employee/campaign.scss

```
@@ -1379,4 +1380,127 @@
       box-shadow: 0px 2px 7px 0px #00000026;
     }
   }
+}
+
+.reports-configuration {
+  &__heading {
+    font-family: var(--digitv2-fontFamily-rc);
+    font-weight: var(--digitv2-fontWeight-bold);
+    font-style: Bold;
```

Stylelint (17.13.0)

[error] Expected "Bold" to be "bold" (value-keyword-case)

(value-keyword-case)